vocode/streaming/telephony/server/base.py

```diff
@@ -95,12 +96,21 @@ def __init__(
                 methods=["POST"],
             )
         # vonage requires an events endpoint
-        self.router.add_api_route("/events", self.events, methods=["GET"])
+        self.router.add_api_route("/events", self.events, methods=["GET", "POST"])
         self.logger.info(f"Set up events endpoint at https://{self.base_url}/events")
 
+        self.router.add_api_route("/recordings/{conversation_id}", self.recordings, methods=["GET", "POST"])
+        self.logger.info(f"Set up recordings endpoint at https://{self.base_url}/recordings/{{conversation_id}}")
+ 
     def events(self, request: Request):
         return lambda: Response()
 
+    async def recordings(self, request: Request, conversation_id: str):
+        recording_url = (await request.json())["recording_url"]
+        if self.events_manager is not None and recording_url is not None:
+            self.events_manager.publish_event(RecordingEvent(recording_url=recording_url, conversation_id=conversation_id))
+        return lambda: Response()
```

Comment:
can we make this `return Response()` and change the `add_api_route` to be `self.router.add_api_route("/recordings")(self.recordings)` - I think that should work?

the events endpoint is indeed written in an idiosyncratic way, my bad 😅

Reply:
done